__tests__/app.test.js

```diff
@@ -303,4 +303,26 @@ describe('GET /api/users', ()=>{
             expect(body.msg).toBe("Route not found")
         })
     })
+})
+
+describe('GET /api/articles?topic=cats', ()=>{
+    test('should filter responses based on the topic added in the query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles.length).toBe(1)
+        })
+    })
+})
+
+describe('GET /api/articles?topic=cats', () =>{
```

Comment:
You can and should combine the this and the above describe block as they are the same endpoint